applications/OptimizationApplication/custom_utilities/response/mass_response_utils.h

```diff
@@ -0,0 +1,81 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
```

Comment:
@sunethwarna Thank you for your work on the implementation. I was wondering if you could explain the major benefits of having response utilities instead of response function in C++. Is it primarily about parallelization? When I review the implementation, it seems that everything is the same as the implementation we currently have under custom responses, with only namings changed. I am having difficulty understanding the reason for this change. Would you be willing to discuss this with me further?

Reply:
As we discussed, main reasoning to have python response function and supporting utilities in cpp is to following reasons:
1. Having python response function allows developers to use both python level (i.e. `AnalysisStage`, etc) and `CPP` level developments without any problems.
2. Utilities are not meant to have object with life, therefore, they are made `static`. Hence, all the inputs and outputs are passed throught the function signature (no hidden tasks). This will be easier for a developer to follow than following two objects with life in python and c++.